lib/signers/presign.js

```diff
@@ -24,7 +24,11 @@ function signedUrlBuilder(request) {
         code: 'InvalidExpiryTime', message: message, retryable: false
       });
     }
-    request.httpRequest.headers[expiresHeader] = expires;
+
+    var parsedUrl = AWS.util.urlParse(request.httpRequest.path);
+    var queryParams = parsedUrl.search
+      ? AWS.util.queryStringParse(parsedUrl.search.substr(1)) : {};
+    queryParams['Expires'] = expires;
```

Comment:
It doesn't look like queryParams is used after it's created.

Reply:
Will remove.